docs/src/guides/wordpress/composer/migrate.md

```diff
@@ -6,107 +6,129 @@ description: |
+   To safely make changes to your repository and Platform.sh environment, run the following command:
 
-    ```bash
-    $ git checkout -b composer
-    ```
-3. **Initialize repository as Composer project**
+   ```bash
+   $ git checkout -b composer
+   ```
 
-    You need to have a `composer.json` on your project to use Composer. This file defines the dependencies you want to include (WordPress itself, plugins, and themes). When those dependencies are installed, a matching `composer.lock` file is generated from it that locks each of them down to the minor version, ensuring repeatable builds until you update.
-    
-    Run the command:
+2. Turn your repository into a Composer repository.
+
+   To use Composer, you need:
+
+   - A `composer.json` file listing all the dependencies needed for your project to run 
+     (WordPress itself, its plugins, and its themes).
+   - A `composer.lock` file listing the exact versions of all the dependencies installed on your project.
+     Generated from the `composer.json` file, it ensures repeatable builds until you update.
+
+   To turn your repository into a Composer repository and generate those files, run the following command:
 
-    ```bash
-    $ composer init
-    ```
+   ```bash
+   $ composer init
+   ```
 
-    This will prompt you to set some metadata attributes for the project, such as its name and license information. When you get to the part about installing dependencies, type `no`, as you will add them in a later step.
+   When prompted, set metadata attributes for your project, 
+   such as its name and license information.
+   When you get to the part about installing dependencies, type `no`, 
+   as you will add them later.
 
-4. **Cleanup WordPress core**
+3. Clean up WordPress core
 
-    Your existing installation of WordPress core is assumed to be in some subdirectory of your repository, such as `wordpress`. When we start managing WordPress with Composer in the next step, *all code in this subdirectory is overwritten*. The [previous guide](/guides/wordpress/vanilla/_index.md) instructs you to move all plugins and themes out of WordPress core and into a separate `plugins` subdirectory, so if you have not already done so consult that guide and do so now. 
+   Your existing installation of WordPress core is assumed to be in a subdirectory of your repository (often named `wordpress`).
+   For Composer to manage WordPress, this subdirectory needs to be overwritten.
+   Make sure that you move all your important data to another subdirectory.
+   For instance, add your plugins and themes to a separate `plugins` subdirectory. 
 
-    Inspect WordPress core carefully, and ensure that their isn't any data in that subdirectory you would not like to lose specific to your site. Unless you are managing WordPress core as a [submodule](/development/submodules.md#removing-submodules), you can now remove that subdirectory 
+   You can now remove the `wordpress` subdirectory: 
```

Comment:
But probably any change like that should go at the start of the previous paragraph, since that wouldn't apply either? At least somewhere in this section it would probably be nice.